### ISSUE TITLE
kubefate crashed when chart incorrect

### ISSUE THREAD

**Issue by asdfsx** [FederatedAI/KubeFATE#385]
**To Reproduce**
upload a helm chart which format is incorrect.
Then use the chart to install a new cluster

**What happen?**
kubefate crashed

**Screenshots**
![image](https://user-images.githubusercontent.com/2292716/112926728-71ec9380-9146-11eb-8d06-37ab2e16385a.png)

**Additional context**
`template.Must` will panic when err is not nil. I think it should be replaced.
```
func Must(t *Template, err error) *Template {
	if err != nil {
		panic(err)
	}
	return t
}
```

**Comment by LaynePeng** [comment]
Thanks for the raising it. @owlet42 could you have a look at it.

**Comment by owlet42** [comment]
fixed by #386
